content/data_science/3_introduction_to_statistics/descriptive_statistics/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+_db_id: 109
+title: descriptive statistics
+---
+
+# What is Descriptive Statistics 
+
+Descriptive statistics, a field within statistics, encompasses the gathering, analysis, interpretation, and presentation of data with the aim of summarizing and delineating its primary characteristics.Its primary purpose is to provide a concise and meaningful representation of a dataset, offering insights into its central tendencies, variability, and distribution. Here are key concepts within descriptive statistics we are going to cover in this course:
+
```

Comment:
This topic seems a bit like a waste of time.

Think about the learner.

The learner will start the card. Read the content (less than one minute), wonder what these things are, scratch their head a bit. and close the card. 

They are unlikely to benefit from this kind of thing. 

Each topic should be helpful in some way. Are these concepts introduced properly somewhere else? 

Or is there a better way to describe descriptive statistics?